### PR TITLE
feat(odiff): replicate pixelmatch alpha option

### PR DESCRIPTION
**Pull request by JonasBa**
When generating diffs, we want to expose the diffed pixels compare to the rest of the images (pixelmatch was doing that), but we lost the functionality when migrating to odiff. This PR reintroduces that functionality by multiplying the source image with a white transparent color and overlaying the diff mask.

### CODE REVIEW

File: src/util/getDiffODiff.ts [hunk 1]
```diff
@@ -20,21 +25,53 @@ export async function getDiffODiff(
...
 
-  const diff = await compare(file1, file2, diffPath, {
+  const OPTIONS: ODiffOptions = {
     antialiasing: false,
     failOnLayoutDiff: false,
     outputDiffMask: false,
     threshold: 0.1,
     ...options,
-    // @ts-ignore
+    // @ts-ignorew
```
Comment (billyvg) [inline src/util/getDiffODiff.ts:34]:
```suggestion
    // @ts-expect-error
```

**Review by billyvg — COMMENTED** [review]
Can we see what `__tests__/util/diffSnapshots/imgs/diff/alpha-diff.png` looked like with pixelmatch? is it similar?

**Comment by JonasBa** [comment]
@billyvg yeah, let me checkout master and copy this test so we can see what pixelmatch shows. It's very much the same, minus the antialiasing

**Comment by JonasBa** [comment]
@billyvg this is what the diff looks like (same test using pixelmatch on master)

alpha-diff

**Review by billyvg — APPROVED** [review]

**Pull request merged**